recipes/module_imap.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Author:: Artur Melo (<artur.melo@beubi.com>)
+# Cookbook Name:: php
+# Recipe:: module_imap
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+pkg = value_for_platform(
```

Comment:
As nice as value_for_platform is this probably isn't the best place to use it.  Since you can't give a version range you'll forever be updating this for the latest version of RHEL.  Right now 5 is up to 5.11 so this won't work on fully patched systems.  Instead you might want to turn this into something like this

if node.platform_family == 'rhel' && node.platform_version.to_i == 5
  pkg = 'php-imap'
else
 pkg = 'php5-imap'
end